app/controllers/application_controller.rb

```diff
@@ -13,4 +13,12 @@ def render_not_found_response(exception)
   def render_unprocessable_entity_response(exception)
     render json: { error: exception.message }, status: :unprocessable_entity
   end
+
+  def set_locale
```

Comment:
Because  the email delivery is  made in another thread, different from the request, maybe setting the locale in the  request could make to not be configured correctly in the mail thread, maybe you can change the default in the mailer

Reply:
Done